apps/server/src/utils/coerceType.ts

```diff
@@ -40,3 +58,35 @@ export function coerceNumber(value: unknown): number {
   }
   return parsedValue;
 }
+
+/**
+ * @description Converts a value to a colour if possible, throws otherwise
+ * @param {unknown} value - Value to be converted to a colour.
+ * @returns {string} - The converted value as a string.
+ * @throws {Error} Throws an error if the value is null or undefined.
+ */
+const hexColor = new RegExp(/^#((?:[a-f\d]{1,2}){3,4})$/, 'i');
```

Comment:
we like tests for regex :D 
Maybe create a function isRegex and test that? we have somewhere in the app a regex file with a few useful regexes

Reply:
the regex file is in the client 
but I added `packages\utils\src\isColourHex.ts` feel free to move or rename it